CHANGELOG.md

```diff
@@ -7,12 +7,22 @@ For older changes see the [archived Singularity change log](https://github.com/a
 
 ## Changes for v1.4.x
 
-## Changes for v1.3.x
+## v1.3.2 - \[2024-05-28\]
```

Comment:
I would leave the `## Changes for v1.3.x` in the main branch, but it's not worth redoing the PR, we'll add it back with the next change destined for 1.3.x.